flutter/cpp/datasets/ade20k.cc

```diff
@@ -213,7 +213,14 @@ float ADE20K::ComputeAccuracy() {
   }
   float iou_sum = 0.0;
   for (int j = 0; j < num_classes_; j++) {
-    auto iou = tp_acc_[j] * 1.0 / (tp_acc_[j] + fp_acc_[j] + fn_acc_[j]);
+    auto sum = tp_acc_[j] + fp_acc_[j] + fn_acc_[j];
+    if (sum == 0) {
+      // in our integration test we use very small dataset
+      // which doesn't have some of the classes
+      iou_sum += 1.0;
+      continue;
+    }
+    auto iou = tp_acc_[j] * 1.0 / sum;
```

Comment:
@freedomtan Please check if this change has an impact on the submission, since the model / backend can achieve higher accuracy by just ignoring some classes.

Reply:
Backends can't use this because the `if (sum == 0)` branch is triggered only when _dataset_ doesn't have some classes.
And this change doesn't really affect us because if you mess with datasets in an appropriate way, you can reach arbitrary high accuracy, regardless of this change.

Still I updated the code to make accuracy lower when dataset doesn't contain some classes, instead of adding `+1` for missing classes.

Reply:
Since the updated code doesn't increase accuracy anymore, it should be ok now.